controlplane/eks/api/v1beta1/awsmanagedcontrolplane_types.go

```diff
@@ -165,6 +165,10 @@ type AWSManagedControlPlaneSpec struct { //nolint: maligned
 	// +optional
 	OIDCIdentityProviderConfig *OIDCIdentityProviderConfig `json:"oidcIdentityProviderConfig,omitempty"`
 
+	// AccessConfig specifies the access configuration information for the cluster
+	// +optional
+	AccessConfig *AccessConfig `json:"accessConfig,omitempty"`
```

Comment:
You only need to add new API types to the latest API version (so v1beta2) in this case. This can be removed.

Due to the roundtripping requirement only having the field in the latest version of the API will probably cause the  "fuzzing" test to fail. So you'll need to store & store the new API fields. Feel free to ping here or in slack if you need a hand.

Reply:
Yeah i originally had that but yes the fuzzing test was failing, and it was unclear how to get the fuzzer to ignore that field for roundtrip.